metaflow/sidecar_messages.py

```diff
@@ -22,7 +22,11 @@ def serialize(self):
 
 
 def deserialize(json_msg):
+    dict = {
+        1: MessageTypes.SHUTDOWN,
+        2: MessageTypes.LOG_EVENT
+    }
     parsed_json_msg = json.loads(json_msg)
-    return Message(MessageTypes(parsed_json_msg['msg_type']),
+    return Message(dict[parsed_json_msg['msg_type']],
```

Comment:
dict is not a great name :).

Reply:
haha goodpoint. That was actually just meant to be temporary. I replaced MessageTypes with IntEnum so the dict is no longer needed